Update readme (#26)
BlueMirrors/cvu: README.md

<br>

> Computer Vision deployment tools for dummies and experts.<br><br>

Reviewer: ```suggestion

Computer Vision deployment tools for dummies and experts.<br><br>
```

Reviewer: Image alignment seems to be off without this extra line after ```<br>``` (checked on chrome and safari)

Whether you are developing an optimized computer vision pipeline or just looking to use some quick computer vision in your project, CVU <img src="https://raw.githubusercontent.com/BlueMirrors/cvu/master/static/logo.png" width="12"> can help! Designed to be used by both the expert and the novice, CVU <img src="https://raw.githubusercontent.com/BlueMirrors/cvu/master/static/logo.png" width="12"> aims at making CV pipelines easier to build and consistent around platforms, devices and models.<br><br>

```bash
pip install cvu-python
```

# Index 📋

- [Getting Started](#cvu--says-hi) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1FvebFw40Bm0bUHWCgS0-iuYp8AKLIfSh?usp=sharing)

> - [What and why is CVU?](https://github.com/BlueMirrors/cvu/wiki)

Reviewer 2: ```suggestion
- [What is and why CVU?](https://github.com/BlueMirrors/cvu/wiki)
```
Even my suggestion might be wrong but ```Why is CVU ? ``` does not look correct. right?

Author: Now that you mention it, I think it looks weird.

Reviewer: How about we simply change it to "Introducing CVU"? Or just keep either "What is CVU" or "Why CVU".

> - [What and why is CVU?](https://github.com/BlueMirrors/cvu/wiki)

Reviewer 2: Same suggestion here.

Author: Let's say we will change.

[Index](#index-)

> How much installation-steps and code will you need to run object detection on a video with a TensorRT backend? How complicated is it be to test that pipeline in Colab?<br><br>

Reviewer 2: ```suggestion
How many installation-steps and lines of code will you need to run object detection on a video with a TensorRT backend? How complicated is it be to test that pipeline in Colab?<br><br>
```

With CVU <img src="https://raw.githubusercontent.com/BlueMirrors/cvu/master/static/logo.png" width="12">, you just need the following! No extra installation steps needed to run on Colab, just pip install our tool, and you're all set to go!<br>

> 

Reviewer: Should we mention that we're using ```vidsz``` (and possibly add a link) in the demo, and it's optional?

Reviewer 3: Yes, I think its a good idea.. to get developers to stop coding open-cv boiler plate stuff
